torch/_dynamo/optimizations/backends.py

```diff
@@ -556,6 +558,10 @@ def _wrapped_bw_compiler(*args, **kwargs):
 
     bw_compiler = kwargs.get("bw_compiler") or kwargs["fw_compiler"]
     kwargs["bw_compiler"] = _wrapped_bw_compiler
+    if "shape_env" in kwargs:
+        from functorch._src import config as functorch_config
+
+        functorch_config.use_dynamic_shapes = True
```

Comment:
this is pretty sketch

Reply:
This is just for prototype. The real way is to call patch